tests/Cargo.toml

```diff
@@ -36,7 +36,7 @@ yktracec = { path = "../yktracec", features = ["yk_testing"] }
 
 [dev-dependencies]
 criterion = { version = "0.5.1", features = ["html_reports"] }
-lang_tester = "0.7.4"
+lang_tester = { version = "0.7.4", path = "../../lang_tester" }
```

Comment:
You'll need to make this point to the branch in Edd's github fork. Cargo has the settings you need to do this.

Reply:
Hold on, we don't want to merge that into `yk/master`. It fudges lang_tester to not match stdin/stdout.

What's the plan here?